modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -260,7 +260,7 @@ export function querystring(params: HTTPQuery, prefix: string = ''): string {
         .join('&');
 }
 
-function querystringSingleKey(key: string, value: string | number | null | boolean | Array<string | number | null | boolean> | Set<string | number | null | boolean> | HTTPQuery, keyPrefix: string = ''): string {
+function querystringSingleKey(key: string, value: string | number | null | undefined | boolean | Array<string | number | null | boolean> | Set<string | number | null | boolean> | HTTPQuery, keyPrefix: string = ''): string {
```

Comment:
This is the only change, the rest was generated automatically.

Listing `undefined` under the types allowed for the value argument of querystringSingleKey is necessary because when noUncheckedIndexedAccess is enabled in tsconfig.json, `params[key] could be undefined`.